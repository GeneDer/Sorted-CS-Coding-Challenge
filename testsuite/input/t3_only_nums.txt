9*9 8)98! 6@@6#1 3%78^1 9&82% 1^1 %%2 $% 3% $0 -$#%999 9%$27 10$#01 -2#$71 9%8
