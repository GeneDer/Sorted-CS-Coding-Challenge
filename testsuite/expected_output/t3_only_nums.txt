-999 -271 0 2 3 11 98  99 661 898 927 982 1001 3781